types/check-types/index.d.ts

```diff
@@ -159,6 +160,7 @@ interface CheckType {
     arrayLike: ArrayLikeFunction;
     iterable: IterableFunction;
     includes(a: any[], value: any): boolean;
+    in(value: any, a: any[]): boolean;
```

Comment:
In the library documentation you can use `in` function to check for value in array, object and string

https://gitlab.com/philbooth/check-types.js

Your type definition states that second argument is array only.

It will be nice also that you add test code to mirror the possibility to use `in` with object, string.